### ISSUE TITLE
Asset Manager can't load .esm Modules

### ISSUE THREAD

**Issue by wernerjoss** [getgrav/grav#3783]
I just stumbled over a Problem with Javascript esm Module loading via Asset Manager as described in [the docs](https://learn.getgrav.org/17/themes/asset-manager#addjsmodule-asset-options) - it turned out that it looks like being impossible to do this.
my example was [photoswipe slideshow](https://github.com/htmltiger/photoswipe-slideshow), which is only provided as an esm Module.
I tried with several Options (loading...) but none worked.
So, I guess, this is a feature request.

**Comment by rhukster** [comment]
Can you provide an example of the twig you are using and what you expect the output to be?

**Comment by wernerjoss** [comment]
yes, sure - in twig, I can load the desired esm Modules like so:

`<script type="module">
				import PhotoSwipeLightbox from 'https://unpkg.com/photoswipe@5.4.2/dist/photoswipe-lightbox.esm.js';
				import PhotoSwipe from 'https://unpkg.com/photoswipe@5.4.2/dist/photoswipe.esm.js';
				import PhotoSwipeSlideshow from 'https://cdn.jsdelivr.net/gh/htmltiger/photoswipe-slideshow/photoswipe-slideshow.esm.min.js';
				const lightbox = new PhotoSwipeLightbox({ gallery: '#photoswipe-gallery-id',  pswpModule: PhotoSwipe, children: 'a'});
				const slideshowPlugin = new PhotoSwipeSlideshow(lightbox,6); // statt 4 sec
				lightbox.init();
</script>
`
which seems to work, if used in a theme's base.html.twig, as it is just pure html.
my intention, however, is to load these in a plugin (PhotoswipeExtension.php), like so:

`
sprintf('import PhotoSwipeLightbox from "https://unpkg.com/photoswipe@5.4.2/dist/photoswipe-lightbox.esm.js";'),
sprintf('import PhotoSwipe from "https://unpkg.com/photoswipe@5.4.2/dist/photoswipe.esm.js";'),
sprintf('import PhotoSwipeSlideshow from "https://cdn.jsdelivr.net/gh/htmltiger/photoswipe-slideshow/photoswipe-slideshow.esm.min.js";'),
sprintf('const lightbox = new PhotoSwipeLightbox({ gallery: "#photoswipe-gallery-id",  pswpModule: PhotoSwipe, children: "a"});'),
sprintf('const slideshowPlugin = new PhotoSwipeSlideshow(lightbox,6);'),
sprintf('lightbox.init();'),
 `
but this simply does nothing, no modules loaded visible in debugger.
same happens when I copy the modules from upkg to local file system and try to load from there.
